eclair-core/src/main/scala/fr/acinq/eclair/db/DualDatabases.scala

```diff
@@ -55,360 +59,369 @@ object DualDatabases extends Logging {
         throw t
     }
   }
+
+  def getDatabases(dualDatabases: DualDatabases): (SqliteDatabases, PostgresDatabases) =
+    (dualDatabases.primary, dualDatabases.secondary) match {
+      case (sqliteDb: SqliteDatabases, postgresDb: PostgresDatabases) =>
+        (sqliteDb, postgresDb)
+      case (postgresDb: PostgresDatabases, sqliteDb: SqliteDatabases) =>
+        (sqliteDb, postgresDb)
+      case _ => ???
```

Comment:
Can you explicitly throw with an error message here instead?

Reply:
Done with 5c3eac546.